src/cldr/load.ts

```diff
@@ -1,15 +1,21 @@
 // required for Globalize/Cldr to properly resolve locales in the browser.
-import 'cldr/unresolved';
+import 'cldrjs/dist/cldr/unresolved';
+import { Require } from 'dojo-interfaces/loader';
 import Map from 'dojo-shim/Map';
-import global from 'dojo-core/global';
 import load from 'dojo-core/load';
 import coreRequest, { Response } from 'dojo-core/request';
 import has from 'dojo-has/has';
 import Promise from 'dojo-shim/Promise';
-import * as Globalize from 'globalize/globalize';
+import * as Globalize from 'globalize/dist/globalize';
```

Comment:
It does, and I found a way to make this work; I just need to update the Globalize typings first.